install_dependencies.sh

```diff
@@ -32,7 +32,29 @@ echo "Installing Misc. Utilities"
 echo "================================================================"
 
 sudo apt-get install -y\
-    clang-format
+    clang-format\
+    python-rosinstall
+
+
+echo "================================================================"
+echo "Installing Project Dependent ROS packages."
+echo "================================================================"
+
+# Setup rosinstall
+sudo mkdir -p /usr/share/ros/
+sudo chmod a+rwx /usr/share/ros
+rosinstall .
+echo "bash $CURR_DIR/setup.sh" >> ~/.bashrc
```

Comment:
Isn't this line added in `get_started.sh`?

Reply:
Removed